htlcswitch/mailbox.go

```diff
@@ -66,6 +66,14 @@ type MailBox interface {
 	// Reset the packet head to point at the first element in the list.
 	ResetPackets() error
 
+	// SetDustLimits sets the local and remote dust limits so that
+	// DustPackets can be called.
+	SetDustLimits(local, remote lnwire.MilliSatoshi)
```

Comment:
Does this need to be an interface method? Seems like it could be a part of the config otherwise?

Reply:
sure

Reply:
The link doesn't have access to the `memoryMailBox` but instead to the `Mailbox interface`, so needs to be iface method